cds_migrator_kit/records/cli.py

```diff
@@ -55,7 +55,7 @@ def load_records(sources, source_type, eager):
                         'Model missing recid:{}'.format(item['recid']))
                     JsonLogger().add_log(e, output=item)
                 except Exception as e:
-                    cli_logger.warning(e.message)
+                    cli_logger.exception(e)
```

Comment:
@equadon Just out of curiosity what was the error here? I mean why the `KeyError` exception was failing?

Reply:
@zzacharo It failed because of an incompatibility between python 2 and 3. Python 3's standard exceptions no longer have the `message` attribute.